examples/quickstart.rs

```diff
@@ -0,0 +1,62 @@
+use eventstore::{
+    AppendToStreamOptions, Client, Credentials, EventData, ExpectedVersion, ReadResult,
+    ReadStreamOptions,
+};
+use futures::TryStreamExt;
+use serde::{Deserialize, Serialize};
+use std::error::Error;
+use uuid::Uuid;
+
+#[derive(Serialize, Deserialize)]
+struct TestEvent {
+    pub id: String,
+    pub important_data: String,
+}
+
+type Result<A> = std::result::Result<A, Box<dyn Error>>;
+
+pub async fn run() -> Result<()> {
+    // region createClient
+    let settings = "{connectionString}".parse()?;
+    let client = Client::create(settings).await?;
+    // endregion createClient
+
+    // region createEvent
+    let event = TestEvent {
+        id: Uuid::new_v4().to_string(),
+        important_data: "I wrote my first event!".to_string(),
+    };
+
+    let event_data = EventData::json("TestEvent", event)?;
```

Comment:
**SUGGESION:** Minor, but other samples are also setting the event `id` here (ok, at least C# and Java).

Reply:
But the id is optional, if I place it there it becomes mandatory.

Reply:
Why will it become mandatory? It'll be just showing that you can provide it manually.

Reply:
If I move the id upward it becomes mandatory, You can still submit the id by using the `id` function provided by the builder.

Reply:
done